<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTemplate</class>
 <widget class="QDialog" name="AddTemplate">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание нового пользовательского шаблона</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="t_descGBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>601</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Описание шаблона</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="descTextEdit">
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>570</y>
     <width>288</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>Сохранить шаблон как....</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="t_nameGBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Название шаблона</string>
   </property>
   <widget class="QLineEdit" name="t_name_lineEd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Общая информация</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>  Автор:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="author_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Не установлен</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>  Дата создания:  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cdate_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>  Дата изменения:  </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mdate_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="page_orientGBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Ориентация</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QToolButton" name="portretBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/portret.png</normaloff>:/portret.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string>orient_BtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="landscapeBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/landscape.png</normaloff>:/landscape.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string>orient_BtnGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="p_orient_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text">
        <string>Книжная</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="l_orient_label">
       <property name="font">
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text">
        <string>Альбомная</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="marginGBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Поля</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Верхнее</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Левое</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Нижнее</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Правое</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="margin_topSpBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="margin_bottomSBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="margin_leftSpBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="margin_rightSpBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Размер листа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pageSizeCBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="page_sizeGBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>184</y>
     <width>601</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Страницы шаблона</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>16</y>
      <width>581</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="FPnumberChBox_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Лицевая сторона 1-го листа.Экз. №1</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="FPnumberChBox_2">
       <property name="text">
        <string>Лицевая сторона 1-го листа.Экз. №2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="FPnumberChBox_3">
       <property name="text">
        <string>Лицевая сторона 1-го листа.Экз. №3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="FPnumberChBox_4">
       <property name="text">
        <string>Лицевая сторона 1-го листа.Экз. №4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="FPnumberChBox_5">
       <property name="text">
        <string>Лицевая сторона 1-го листа.Экз. №5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="SPnumberChBox">
       <property name="text">
        <string>Лицевая сторона 2-го листа</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="ThnumberChBox">
       <property name="text">
        <string>Обратная сторона каждого листа</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="FvnumberChBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Фонарик</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>t_name_lineEd</tabstop>
  <tabstop>portretBtn</tabstop>
  <tabstop>landscapeBtn</tabstop>
  <tabstop>margin_topSpBox</tabstop>
  <tabstop>margin_leftSpBox</tabstop>
  <tabstop>margin_bottomSBox</tabstop>
  <tabstop>margin_rightSpBox</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTemplate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTemplate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="orient_BtnGroup"/>
 </buttongroups>
</ui>
